.github/workflows/publish.yml

```diff
@@ -76,21 +75,14 @@ jobs:
           NPM_CONFIG_PROVENANCE: 'true'
           NPM_TOKEN: ${{ secrets.NPM_ACCESS_TOKEN }}
 
-      - name: rebase develop with main on publish
-        if: ${{ steps.changesets.outputs.published == 'true' }}
-        run: |
-          git restore .
-          git checkout master
-          git fetch --all
-          git pull origin master
-
-          git checkout develop
-          git restore .
-          git rebase master
-          git push -f
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_CONFIG_PROVENANCE: true
+      - name: Send GitHub Action data to a Slack workflow
```

Comment:
adding a slack hook to send published packages as json to a slack message&channel